site/_resources/envoy.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Envoy compatibility matrix
+layout: page
+---
+
+Due to the agressive deprecation cycle of Envoy's xDS API, not all versions of Contour will work with all versions of Envoy, and vice versa.
+
+This page describes the compatibility matrix of Contour and Envoy versions.
+
+## Supported Envoy versions
+
+|              | Contour 1.0.0 | 
+| ------------ | :-----------:|
+| Envoy 1.11.0 | Not supported<sup>1</sup> |
+| Envoy 1.11.1 | Not supported<sup>2</sup> |
+| Envoy 1.11.2 |  Supported | 
+
+#### Notes
+
+1. [CVE-2019-9512, CVE-2019-9513, CVE-2019-9514, CVE-2019-9515, CVE-2019-9518][1] 
+2. [CVE-2019-15225, CVE-2019-15226][2]
```

Comment:
I was expecting the superscripts to be clickable. How about moving these notes to the end of the doc and adding anchors so the superscripts can link to them?

Reply:
Initially I had the CVE's inline, but the table looked terribly unbalanced. The goal of moving them to notes was to hide the detail from folks who just wanted the TL;DR on which version they should use.

Reply:
I agree that having them as notes make sense. I just kept trying to click on them :)

Reply:
I'm not jazzed about dropping to html for this. Perhaps in the future we can find a Jekyll footnote plugin.

Reply:
Yeh, maybe @SDBrett has some ideas?